content/en/observability_pipelines/guide/run_multiple_pipelines_on_the_same_host.md

```diff
@@ -0,0 +1,126 @@
+    DD_OP_SOURCE_FLUENT_ADDRESS=0.0.0.0:9091
+    DD_OP_DATA_DIR=/var/lib/op-fluent
+    DD_OP_API_ENABLED=true
+    ```
+    In this example:
+    - `DD_OP_DATA_DIR` is set to `/var/lib/op-fluent`, the new data directory you created in the previous step.
+    - `DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_ACCESS_KEY_ID=<redacted>` and `DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_SECRET_ACCESS_KEY=<redacted>` are environment variables for the Datadog Archives Amazon destination. Replace it with the environment variables for your destinations. See [Environment Variables](https://docs.datadoghq.com/observability_pipelines/environment_variables/?tab=sources).
+    - `DD_OP_SOURCE_FLUENT_ADDRESS=0.0.0.0:9091` is the environment variable required for the Fluent source in this example. Replace it with the environment variables for your source. See [Environment Variables](https://docs.datadoghq.com/observability_pipelines/environment_variables/?tab=sources).
+    - `DD_OP_API_ENABLED` is optional. Set it to true if you want to use the `top` or `tap` commands to get more information about your pipeline components. See [Enable Observability Pipelines Worker API](https://docs.datadoghq.com/observability_pipelines/troubleshooting/#enable-the-observability-pipelines-worker-api) for more information.
+1. Create a new systemd service entry, such as `/lib/systemd/system/op-fluent.service`. An example of the content for the entry:
+    ```
+    [Unit]
+    Description="OPW for Fluent Pipeline"
+    Documentation=https://docs.datadoghq.com/observability_pipelines/
+    After=network-online.target
+    Wants=network-online.target
+
+    [Service]
+    User=observability-pipelines-worker
+    Group=observability-pipelines-worker
+    ExecStart=/usr/bin/observability-pipelines-worker run
+    Restart=always
+    AmbientCapabilities=CAP_NET_BIND_SERVICE
+    EnvironmentFile=-/etc/default/op-fluent
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+    - The service name is specific to your use case, `<your_use_case_name>.service`. In this example, the service name is `op-fluent` since the pipeline is using the Fluent source.
+    - The description is for your specific use case, where in this example it is `OPW for Fluent Pipeline`.
+    - `EnvironmentFile` is set to the new systemd service environment variables file you created, which in this example is: `-/etc/default/op-fluent`.
+1. Run this command to reload systemd:
+    ```
+    sudo systemctl daemon-reload
+    ```
+1. Run this command to start the new service:
+    ```
+    sudo systemctl enable --now op-fluent
+    ```
+1. Run this command to verify the service is running:
+    ```
+    sudo systemctl status op-fluent
+    ```
+
+You can run the command `sudo journalctl -u op-fluent.service` to help you debug any issues.
+
+## Deploy the pipeline
+
+1.  Navigate to the second pipeline's Install page.
+1.  In the **Deploy your pipeline** section, you should see your second Worker detected. Click **Deploy**.
```

Comment:
```suggestion
1.  In the **Deploy your pipeline** section, you should see your additional Worker detected. Click **Deploy**.
```
Just suggesting to change the wording here in case some users create 2 or more additional workers.